src/test/java/com/revaturelabs/ask/AskApplicationControllerTests.java

```diff
@@ -299,10 +333,300 @@ public void testCreateQuestion() {
+    exampleQuesiton14.setHighlightedResponseId(1);
+    exampleQuesiton14.setCreationDate(date);
+    exampleQuesiton14.addTagToQuestion(javaTag);
+    exampleQuesiton14.addImageToImages(image);
+    exampleQuesiton14.setImages(null);
+
+    assertFalse(
+        exampleQuesiton.equals(exampleQuastion2) & exampleQuastion2.equals(exampleQuesiton));
+    // assertTrue(exampleQuestion3.equals(exampleQuastion2) &
+    // exampleQuastion2.equals(exampleQuestion3));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton4) & exampleQuesiton4.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton5) & exampleQuesiton5.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton6) & exampleQuesiton6.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton7) & exampleQuesiton7.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton8) & exampleQuesiton8.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton9) & exampleQuesiton9.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton10) & exampleQuesiton10.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton11) & exampleQuesiton11.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton12) & exampleQuesiton12.equals(exampleQuesiton));
+    // assertFalse(exampleQuesiton.equals(exampleQuesiton13) &
+    // exampleQuesiton13.equals(exampleQuesiton));
+    assertFalse(
+        exampleQuesiton.equals(exampleQuesiton14) & exampleQuesiton14.equals(exampleQuesiton));
+    assertFalse(exampleQuesiton.equals(user));
+
+    assertFalse(exampleQuesiton.hashCode() == exampleQuastion2.hashCode());
+
+    assertTrue(
+        exampleQuesiton.getAssociatedTags() != null & exampleQuastion2.getAssociatedTags() != null);
+    assertTrue(exampleQuesiton.getBody() != null & exampleQuastion2.getBody() == null);
+    assertTrue(exampleQuesiton.getHead() == exampleQuesiton4.getHead()
+        & exampleQuesiton.getHead() != null);
+    assertTrue(exampleQuesiton.getHighlightedResponseId() == 1
+        & exampleQuastion2.getHighlightedResponseId() == null);
+    assertTrue(exampleQuesiton.getId() == 1 & exampleQuastion2.getId() == null);
+    assertTrue(exampleQuesiton.getQuestionerId() == 1 & exampleQuastion2.getQuestionerId() == null);
+    assertTrue(exampleQuesiton.getResponses() != null & exampleQuastion2.getResponses() == null);
+    assertTrue(exampleQuesiton.getUser() != null & exampleQuastion2.getUser() == null);
+    assertTrue(exampleQuesiton.getCreationDate() != exampleQuastion2.getCreationDate()
+        & exampleQuastion2.getCreationDate() == null);
+    assertTrue(exampleQuesiton.getImages() != null & exampleQuastion2.getImages() != null);
+    assertTrue(exampleQuesiton.getClass() == exampleQuastion2.getClass());
```

Comment:
@DennisY-H have you considered using some sort of looping structure to accomplish the initialization? It seems oddly wasteful to do it this way. Not only that, but you could have created your mock data in one central place and just reused it in each test.